packages/web/src/components/codeMirrorWrapper.tsx

```diff
@@ -27,9 +27,16 @@ export type SourceHighlight = {
   type: 'running' | 'paused' | 'error';
 };
 
+export enum Language {
+  javascript = 'javascript',
+  python = 'python',
+  java = 'java',
+  csharp = 'csharp'
+}
+
 export interface SourceProps {
   text: string;
-  language: string;
+  language: keyof typeof Language;
```

Comment:
Our preferred way would be:

```js
type Language = 'javascript' | 'python' | 'java' | 'csharp';
```

and then use string literals in the code

Reply:
yes makes it more aligned with the rest, I changed it!